README.md

```diff
@@ -80,77 +80,82 @@ The platform is built on top of [Ambire Wallet](https://www.ambire.com) and uses
 
-Campaigns are created with a total budget (e.g. 5000 USDC) and a specification of the desired result: e.g. purchase as many impressions as possible for this ad, with a maximum allowed price per impression and targeting information.
-
-Because campaigns represent a financial commitment on a smart contract, they can be also be seen as smart, automated insertion orders.
+#### Layer 2
 
-The cryptocurrencies that can be used for a campaign depend on what [Core](#core) is used and what it supports: e.g. the Ethereum implementation supports all ERC20 tokens, but it can also be deployed on any EVM network.
+Layer 2 refers to blockchain scaling solutions, which allow financial transactions or other state transitions to happen very fast, off the blockchain, while still being enforceable or eventually being committed to the underlying blockchain. Ideally, Layer 2 solutions allow throughput levels seen in centralized systems, while still being as trustless and censorship-resistant as blockchains. In AdEx, we use a scaling primitive, called OUTPACE, which allows for multiple campaigns to run on the same payment channel, thus allowing for lower transaction fees.
 
-In the AdEx protocol, each campaign uses a payment channel powered by a technology we call **OUTPACE**. Multiple campaigns can run on the same payment channel, as payment channels are created between **validators** (more on this later).
+#### Validators
+Validators in the AdEx ecosystem are responsible for verifying and validating transactions on the blockchain, ensuring the security and transparency of the network. More about validators can be found here. 
 
-#### Layer 2
+#### Observers
+The observers are delegated to collect events in relation to a certain campaign. All validators of a campaign are, by definition, observers of all events related to it.
 
-Layer 2 refers to blockchain scaling solutions, which allow financial transactions or other state transitions to happen very fast, off the blockchain, while still being enforceable or eventually being committed to the underlying blockchain.
 
-Ideally, layer 2 solutions allow throughput levels seen in centralized systems, while still being as trustless and censorship-resistant as blockchains.
+### Layer 2
 
-In AdEx, we use two scaling primitives that we defined: **OCEAN** and **OUTPACE**.
+#### OUTPACE
 
-#### Offchain unidirectional trust-less payment channel (OUTPACE)
+In the AdEx protocol, each campaign uses a payment channel powered by a technology we call OUTPACE. Multiple campaigns can run on the same payment channel, as payment channels are created between validators.
```

Comment:
there are OUTPACE docs here, https://github.com/AmbireTech/adex-protocol/blob/master/OUTPACE.md, you should link them.

To link an internal md file just link it like this "/OUTPACE.md", for example `[OUTPACE](/outpace.md)`